### ISSUE TITLE
MenuBar

### ISSUE THREAD

**Issue by In-vision** [In-vision/Master-Chess#7]
Note: this menuBar will be in the game itself (when playing chess already), it will not be accesible through our other interfaces (for now).
Presently, the application is missing some key parts like a new game, restart game, save game (we will have a little back-end for the project, we will keep up to three games saved). We don't want the About part, since we will have an interface of the main menu where we will establish our information.

Ideally, if we want to have three games saved, that means that every time we click on the new game of the menuBar option, it will return to the initial interface where we can choose what game we will override our playing.

So, to summarize:
New game: return to initial interface of the saved games.
Restart game: return the game to it's original state and keep overriding the information of that game in the same file.
Save game: self-explanatory, save the game in that file.

To make the project more complete, we will add the feature of returning to the main menu.
